postgres/archiver-restic.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: archiver.kubedb.com/v1alpha1
+kind: PostgresArchiver
+metadata:
+  name: postgresarchiver-sample
```

Comment:
May be `pg-arch-restic`. 
The other one will be `pg-arch-vsc`

Reply:
updated